### PR TITLE
Make rawgit branch name customizable

### PR DESCRIPTION
**Pull request by tomalec**
PolymerElements/iron-component-page/issues/101

### CODE REVIEW

**Comment by tjsavage** [comment]
This LGTM - WDYT @justinfagnani?

File: iron-component-page.html [hunk 1]
```diff
@@ -292,6 +292,15 @@
           },
 
           /**
+           * The name of a brnach used to fetch source for gh-pages.
```
Comment (samuelli) [inline iron-component-page.html]: typo for branch

**Comment by tomalec** [comment]
@samuelli thanks, I've fixed the typo

**Comment by tomalec** [comment]
@tjsavage, @justinfagnani is there anything more I can do?
If there is more 2.0/V1 ready branch to rebase to, just late me know.

**Comment by tjsavage** [comment]
Thanks @tomalec ! This looks good to me - @samuelli what do you think?

**Pull request merged**